EnglishFilter/sections/specific.txt

```diff
@@ -291,6 +291,7 @@ techpowerup.com##.site-headr__logo-wrapper + * a[href*="/images/"] > img
 techpowerup.com##div[class*="bar-item"] a[href*="/images/"] > img
 techpowerup.com#%#//scriptlet('remove-attr', 'wdrwgbethte', 'div')
 !
+scotthyoung.com##.ultra-banner-container
```

Comment:
their own self-promo

Reply:
So it should be in annoyances self promo?

Reply:
In general - yes, but it is not annoying.